components/webext-storage/src/schema.rs

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// XXXXXX - This has been cloned from places/src/schema.rs, which has the
+// comment:
+// // This has been cloned from logins/src/schema.rs, on Thom's
+// // wip-sync-sql-store branch.
+// // We should work out how to turn this into something that can use a shared
+// // db.rs.
+//
+// And we really should :) But not now.
```

Comment:
I look forward to growing even more levels of inception here in future 😆

Reply:
[oops - re-adding this comment to the right place!]

:) I think schema management, and maybe even db management in general, is somewhere that remerge-v2 might be able to help with.